src/makeRobotAccountant.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const result = (x) => {
+    count++;
+
+    if (count > 3) {
+      count -= 2;
+
+      return () => {
+        return 'Bzzz... Error!';
+      };
+    }
+
+    if (count <= 3) {
+      return (y) => x + y;
+    }
```

Comment:
```suggestion
    return (y) => x + y;
```
Since you have `return` in prev if, you don't need if here